packages/mdc-icon-button/_mixins.scss

```diff
@@ -144,18 +144,26 @@
   }
 }
 
+///
+/// Sets the font color and the ripple color to the provided color value for an enabled icon button.
+/// @param {Color} $color - The desired font and ripple color.
+///
 @mixin mdc-icon-button-ink-color($color, $query: mdc-feature-all()) {
-  $feat-color: mdc-feature-create-target($query, color);
+  @include mdc-icon-button-ink-color_($color, $query: $query);
```

Comment:
Do we not need enabled selector for this?

Reply:
It was never there beforehand, I assume people didn't run into problems with this?

Reply:
Ok. I think we can remove "for an enabled icon button." from docs.

Reply:
Sounds good